openedx/core/djangoapps/programs/utils.py

```diff
@@ -673,12 +673,19 @@ def get_certificates(user, extended_program):
         certificates.append({
             'type': 'program',
             'title': extended_program['title'],
-            'url': program_credentials[0]['certificate_url'],
+            'url': get_program_certificate_url(program_credentials[0]['certificate_url']),
```

Comment:
Is this url only shown to learners for their own certs?

Reply:
yes